src/musicbrainz/tagfetcher.cpp

```diff
@@ -351,6 +351,8 @@ void TagFetcher::slotCoverArtArchiveLinksTaskNetworkError(
         QNetworkReply::NetworkError errorCode,
         const QString& errorString,
         const mixxx::network::WebResponseWithContent& responseWithContent) {
+    Q_UNUSED(errorCode);
+    Q_UNUSED(errorString);
```

Comment:
There is an open TODO some lines below - I'm not sure if we should silence the warning in this case.

Reply:
I don't understand. We should keep the warning as a reminder for a TODO?

Reply:
I'd rather have an open GitHub issue for this instead of leaving warnings in the code to make it more likely to find this `TODO`.

Reply:
At least there should be a comment, that describes that this silences a real warning, and not a false positive.

Reply:
I'll move the TODO up below Q_UNUSED

Reply:
and filed #11903